packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,331 @@
+- This card needs to list the various details of the cluster. Each detail will have a label and a value, for example: name // cluster-name
+- This looks like it calls for a two-column list of the data. One column for the label, and one for the value.
+
+“How much data will this card contain?”
+- In this particular instance of the storage dashboard, the list only contains four items. Not a very large list.
+
+“Is the amount of information in the card static or dynamic?”
+- The items in the card may be added or removed depending on user or stakeholder feedback, but they aren’t changed unless we decide to change them. Therefore, this information is static and we can expect this card to always contain the same amount of information.
+
+Conclusion
+- This card will always contain a two-column list of four items.
+- This card can be small.
+
+**Performance card**
+“What format would most clearly display this information?”
+- This card will display the current performance metrics for five different values. It must show the historical data of the values over time.
+- Show the current value in text and a line graph displaying the value over time. This means each of the five metrics will have a text label, a text value, and a line graph.
+
+“How much data will this card contain?”
+- Mostly answered above. It will contain five metrics, each of which reflecting a current data value and a graph showing the value over time.
+
+“Is the amount of information in the card static or dynamic?”
+- This card will contain metrics that always show the same thing: A label for the metric, a current value, and the value over time. This information itself will change constantly but the amount of or density of information will not. Therefore, it is static.
+
+Conclusion
+- Include five line charts. This card will probably be large.
+
+**Events card**
+“What format would most clearly display this information?”
+- This card will display a list of events as they occur.
+- Each event list item will show a few lines of information: time occurred, event source, event description.
+- This information will be best displayed in a list view format.
+
+“How much data will this card contain?”
+- Events happen fast, and in large numbers. This card could contain thousands of events. The user will need to be able to filter this view to tailor it to their specific needs.
+
+“Is the amount of information in the card static or dynamic?”
+- As this is a list that could contain 0 events or could contain thousands, it is indeed dynamic.
+
+Conclusion
+- This card could contain a massive amount of events. We’ll need to give the card a max height based on the amount of items we think would be reasonable to see at once. Four items at once should be reasonable to set for our max height. The card will then scroll if the user wants to see more events.
+- Since we have a max height of four items and this list view only needs one column, we can estimate that a medium sizing should do.
```

Comment:
Reading this, I think it might seem unclear to users what a small vs medium vs large card looks like - wonder if we could add more guidance around that, or link to card examples/guidelines?